linux-static/linux-static-arm64.Dockerfile

```diff
@@ -1,3 +1,29 @@
-# Using https://github.com/rust-cross/rust-musl-cross
+FROM debian:bookworm
 
-FROM messense/rust-musl-cross:aarch64-musl
+ENV PATH=/root/.cargo/bin:/opt/cross/bin:$PATH
+
+RUN apt-get update && \
+    apt-get install -y build-essential curl file git bison flex
+
+COPY ./aarch64-musl-config.mak /tmp/config.mak
+
+# Build cross-compiling toolchain using https://github.com/richfelker/musl-cross-make,
+# https://github.com/rust-cross/rust-musl-cross.
+# See also `../cross/linux-musl-arm.Dockerfile`.
+RUN cd /tmp && \
+    git clone --depth 1 https://github.com/richfelker/musl-cross-make.git && \
```

Comment:
It looks like this clones the repo to get the code from the default branch.
It would make sense to rely on a tag here? To make builds idempotent on the long term.

Reply:
yes, absolutely, good catch